docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -186,6 +193,45 @@ Module not found: Can't resolve 'aws-crt' in 'xxx'
 
 These errors will not affect the use and functionality of the client, nor will errors appear in the browser console. This is a known transitive issue when using the AWS SDK. The issue is currently tracked [here](https://github.com/aws-amplify/amplify-js/issues/11030).
 
+## Server-Side Rendering (SSR)
+
+If you attempt to use Amplify with server-side rendering, you may encounter the following error:
+
+```
+./node_modules/@aws-amplify/ui-react-core/dist/esm/components/FormCore/FormProvider.mjs
+Attempted import error: 'useForm' is not exported from 'react-hook-form' (imported as 'useForm').
```

Comment:
Would it make more sense to just have this error once, rather than twice on this page?

Reply:
yeah that makes sense. I changed it so the error is a heading under the SSR section and the Next.js section just points here, lmk what you think

Reply:
LGTM